smarts/zoo/worker.py

```diff
@@ -85,6 +89,15 @@ def listen(port):
     parser = argparse.ArgumentParser(
         "Listens for requests to allocate agents and executes them on-demand"
     )
-    parser.add_argument("--port", help="Port to listen on", default=7432, type=int)
+    parser.add_argument(
+        "--port", type=int, default=7432, help="Port to listen on",
+    )
+    parser.add_argument(
+        "--authkey",
+        type=str,
+        default="secret",
```

Comment:
A default at this level is OK.

Reply:
Changed default authentication settings.
1. Running agents in remote server
    + Authentication key is mandatory. 
    + Python command example:
        ```bash
        # Launch agent in remote servers
        $ python3.7 ./smarts/zoo/worker.py --auth_key secret
        # Start Smarts in local host
        $ python3.7 ./examples/single_agent.py scenarios/loop --auth_key secret
        ```
    + scl command example:
        ```bash
        # Launch agent in remote servers
        $ scl zoo worker secret
        # Start Smarts in local host
        $ python3.7 ./examples/single_agent.py scenarios/loop --auth_key secret
        ```
1. Running agents in local host
    + Authentication key is optional.
    + Python command example:
        ```bash
        # No user specified authentication key
        $ python3.7 ./examples/single_agent.py scenarios/loop  
        # User specificed optional authentication key
        $ python3.7 ./examples/single_agent.py scenarios/loop --auth_key secret
        ```
1. Although I have changed the default authentication settings as suggested, I somewhat believe setting default authentication on the remote worker side will be helpful for new users to get started easily without worrying about the various arguments in launching the code.